File: AtlasWeb/Core/Modules/Tibbi_Surec/Kayit_Kabul_Modulu/PatientSearchWithDetails/PatientSearchWithDetailsFormViewModel.ts

import { BaseModel } from 'Fw/Models/BaseModel';
import { SKRSILKodlari } from 'NebulaClient/Model/AtlasClientModel';
import { Type } from 'NebulaClient/ClassTransformer';

export class PatientSearchWithDetailsFormViewModel extends BaseModel {

    public Name: string;
    public Surname: string;
    public MotherName: string;
    public FatherName: string;
    @Type('SKRSILKodlari')
    public CityOfBirth: SKRSILKodlari;
    public DoctorName: string;
    public policlinicName: string;
    public UniqueRefNo: string;
    public Age: string; s;
    public AdmissionPoliclinic: string;
    public Payer: string;
    public AdmissionType: string;
    @Type(() => Date)
    public BirthDate: Date;
    public BirthPlace: string;
    @Type(() => Date)
    public AdmissionDateFirst: Date;
    @Type(() => Date)
    public AdmissionDateSecond: Date;
    @Type(() => Date)
    public AdmissionTimeFirst: Date;
    @Type(() => Date)
    public AdmissionTimeSecond: Date;

    public DispatchNumber: string;
    public DistrictPoliclinicName: string;
    public PatientInServiceName: string;
    public PasaportNo: string;

}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Kayit_Kabul_Modulu/PatientSearchWithDetails/PatientSearchWithDetailsFormViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using TTObjectClasses;
using System.ComponentModel;
using TTInstanceManagement;
using Core.Models;
using System.Collections;

namespace Core.Models
{
    public class PatientSearchWithDetailsFormViewModel
    {
        public string Name
        {
            get;
            set;
        }

        public string Surname
        {
            get;
            set;
        }

        public string MotherName
        {
            get;
            set;
        }

        public string FatherName
        {
            get;
            set;
        }

        public SKRSILKodlari CityOfBirth;
        public string DoctorName
        {
            get;
            set;
        }

        public string policlinicName
        {
            get;
            set;
        }

        public string UniqueRefNo
        {
            get;
            set;
        }

        public string Age
        {
            get;
            set;
        }

        public string AdmissionPoliclinic
        {
            get;
            set;
        }

        public string Payer
        {
            get;
            set;
        }

        public string AdmissionType
        {
            get;
            set;
        }
        public string BirthPlace
        {
            get;
            set;
        }        

        public DateTime BirthDate
        {
            get;
            set;
        }

        public DateTime AdmissionDateFirst
        {
            get;
            set;
        }

        public DateTime AdmissionDateSecond
        {
            get;
            set;
        }

        public DateTime AdmissionTimeFirst
        {
            get;
            set;
        }

        public DateTime AdmissionTimeSecond
        {
            get;
            set;
        }

        public string DispatchNumber
        {
            get;
            set;
        }

        public string DistrictPoliclinicName
        {
            get;
            set;
        }

        public string PatientInServiceName
        {
            get;
            set;
        }

        public string PasaportNo { get; set; }

        public PatientSearchWithDetailsFormViewModel()
        {
        }
      
    }
}